src/android/cordovaSSDP.java

```diff
@@ -74,7 +74,7 @@ public void onFailure(int statusCode, Header[] headers, byte[] responseBody,
 
     public void search(String service, CallbackContext callbackContext) throws IOException {
         final int SSDP_PORT = 1900;
-        final int SSDP_SEARCH_PORT = 1901;
+        final int SSDP_SEARCH_PORT = 12000;
```

Comment:
@psyvision If we removed the fixed port number entirely and call `InetSocketAddress` with 0 I think it will pick a random port. See: https://developer.android.com/reference/java/net/InetSocketAddress#InetSocketAddress(int)

I think that would be preferable and more standard than assigning a fixed port number. Thoughts?

Reply:
It probably would, if it worked :/

I changed the code in my sample application to pass 0 and no devices were detected.  Changing it back to 12000 and the device was immediately picked up.

I'm presuming the net result is when the socket binds each separate time it gets two different ports.  I tried to confirm by logging the port number after binding but both log -1.

Reply:
@psyvision I think you are correct about the two different ports. I found a way to resolve that: https://github.com/konnected-io/cordova-plugin-discovery/pull/2

Can you confirm that this updated version works on your infrastructure? It works for me and uses a different random high port number each time.